src/powerfit_em/powerfitter.py

```diff
@@ -498,26 +498,22 @@ def _cl_rotate_grids(self, rotmat):
                     self._rot_template)
             self._k.rotate_image3d(self._queue, self._gmask, rotmat,
                     self._rot_mask, nearest=True)
-            self._queue.finish()
```

Comment:
As kernels cannot run concurrently (and one will only start after the previous one finished), these `.finish()` calls are not needed as far as I can see.

With some testing, it seems that I can remove all `self._queue.finish()` statements in the inner loop and have the exact same results still :thinking: